aspnetcore/publishing/iis.md

```diff
@@ -459,7 +470,7 @@ Troubleshooting:
 
 * Install the .NET Framework version missing from the system.
 
-* For a framework-dependent deployment, confirm that you have the correct runtime installed on the system. For example if you upgrade a project from 1.0 to 1.1, deploy to the hosting system, and receive this exception, ensure you install the 1.1 framework on the hosting system.
+* For a framework-dependent deployment, confirm that you have the correct runtime installed on the system. For example if you upgrade a project from 1.1 to 2.0, deploy to the hosting system, and receive this exception, ensure you install the 2.0 framework on the hosting system.
```

Comment:
Add a comma after "For example".

Reply:
A pair of prepositional phrases form one pause group, and I think all of the style manuals I've seen call for no comma ... just as if one had two prep phrases anyplace else in the sentence.

> Jane kicked the ball in the house into a wall.

... wouldn't be written ...

> Jane kicked the ball in the house, into a wall.

I think the MS Style Manual dissuades the construction at the start of a sentence. There's no need for it really. I've been modifying my own text lately to stop doing this altogether. Thanks for catching this. This one (and another one earlier in the text) got by me. I'll just kill the first prep phrase to solve it.